oemof_b3/config/settings.yaml

```
@@ -27,6 +27,7 @@ optimize:
   el_gas_relation: electricity_gas_relation  # appears in build_datapackage as well
   el_key: electricity  # prefix of keywords for gas electricity relation
   gas_key: gas  # prefix of keywords for gas electricity relation
+  plot_esys: True  # do you want to creat a graph of esys?
```

```suggestion
  plot_esys: False  # do you want to creat a graph of esys?
```

I propose for the sake of clarity a default to false. Only to be enabled if the dev wants to see it.